models/actions/runner.go

```diff
@@ -52,14 +52,51 @@ type ActionRunner struct {
 	Deleted timeutil.TimeStamp `xorm:"deleted"`
 }
 
-func (r *ActionRunner) OwnType() string {
+// RunnerType defines the runner type
+type RunnerType int
+
+const (
+	// RunnerTypeGlobal defines a global runner
+	RunnerTypeGlobal RunnerType = iota
+
+	// RunnerTypeOrganization defines an organization runner
+	RunnerTypeOrganization
+
+	// RunnerTypeRepository defines a repository runner
+	RunnerTypeRepository
```

Comment:
I guess we already have similar definitions for "project belongs to" last time?

Reply:
`RunnerType` is also added in #24366